packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -1,27 +1,57 @@
+@forward 'settings';
+
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-icon/src/styles/usa-icon';
+@use 'usa-checkbox/src/styles/usa-checkbox';
+@use 'usa-legend/src/styles/usa-legend';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-hint/src/styles/usa-hint';
+@use 'usa-error-message/src/styles/usa-error-message';
+
+@import '../../mixins/uswds-error-border.scss';
+
+:host {
+  display: block;
+}
+
+.usa-checkbox {
+  max-width: 320px;
```

Comment:
This max-width corresponds to the same max-width as `.usa-form` which is what `va-checkbox-group` uses but since individual checkboxes could be used independent of va-checkbox-group, it needed a max-width as well. Otherwise the tiled variation would span across the entire container like this:

![Screenshot 2023-02-22 at 3 21 53 PM](https://user-images.githubusercontent.com/872479/220761825-f3d28ea3-f2ef-44d7-b9b6-6d4773c307a2.png)

I discussed this with Natalie and we thought it would make sense to limit the width of the individuals checkbox components to avoid the full-width effect shown above.